cli/testdataseeder.php

```diff
@@ -219,6 +225,7 @@ function create_standing_data() {
 function create_data_set() {
     create_test_data(XAPI_REPORT_SOURCE_LOG, ROWS);
     create_test_data(XAPI_REPORT_SOURCE_FAILED, ROWS);
+    create_test_data(XAPI_IMPORT_TYPE_HISTORIC, ROWS);
```

Comment:
this should be `XAPI_REPORT_SOURCE_HISTORICAL`

Reply:
I have changed this. The destination table and type is swapped if that had been passed.